client/src/modules/reports/generate/open_debtors/open_debtors.html

```diff
@@ -23,36 +23,101 @@ <h3 class="text-capitalize" translate>REPORT.OPEN_DEBTORS.TITLE</h3>
         <div class="panel-body">
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
 
-
-            <div class="form-group">
+            <div class="form-group" ng-class="{'has-error' : ConfigForm.order.$invalid && ConfigForm.$submitted}">
               <label class="control-label" translate>
-                FORM.PLACEHOLDERS.COLUMN
+                FORM.LABELS.ORDER_BY
               </label>
 
               <ui-select
```

Comment:
This report has at most 6 options for ordering. Do you think we could use a select here? The benefits of a select are cross browser (especially mobile) functionality and simplicity of use.